src/legacy/store/GraphQLDeferredQueryTracker.js

```diff
@@ -491,7 +500,8 @@ function getQueryParentID(
 }
 
 function getRootCallToIDMap(
```

Comment:
how about making this a private instance method instead of passing the `recordStore`? e.g. `_getRootCallToIDMap(query)`